EpiAware/src/EpiObsModels/utils.jl

```diff
@@ -33,11 +33,8 @@ function NegativeBinomialMeanClust(μ, α)
     if isnan(μ) || isnan(α)
```

Comment:
Out of interest, does `nextfloat` etc mean we can get rid of the logic branching here? That would be good from the pov of compilable tape for reverse diff.

Reply:
I don't think so, because `NaN` will not work with clamp. 
```julia
julia> (NaN < 0) | (NaN > 0)
false
```

Reply:
Is your desire to replace the `DiscreteUniform(0, 1_000_000)` with a very wide `NegativeBinomial`? I'm not sure of the right way to do that, but the `clamp` method will not play nice with `NaN`s.